molecule/default/converge.yml

```diff
@@ -45,7 +45,11 @@
 
     - name: Disable certificate checking
       become: yes
-      shell: "sudo -u buildbot git --global http.sslVerify=false"
+      become_user: buildbot
```

Comment:
I think (but not sure), we should drop the "become:" on line 47, and then you dont need the become_user, which is problematic in ansible.  Dont we normally run the buildbot master tasks as the ansible user?

Reply:
And thank you for getting rid of the "shell" task! :)